docs/documentation/01-Getting Started/headless-config.md

```diff
@@ -2,13 +2,33 @@
 slug: /getting-started/headless-config
 sidebar_position: 3
 ---
-# Headless Config
+# Configuring the Framework
 
-The `headless.config.js` file contains several config options for HeadstartWP. This file should export an object of type [HeadlessConfig](/api/modules/headstartwp_core/#headlessconfig).
+The `headstartwp.config.js` (previously, `headless.config.js`) file contains several config options for HeadstartWP. This file should export an object of type [HeadlessConfig](/api/modules/headstartwp_core/#headlessconfig).
+
+## Usage with Next.js
+
+The file **must be named** either `headstartwp.config.js` or `headless.config.js`. When `injectConfig` param of `withHeadstartWPConfig` (previously `withHeadlessConfig`) is set to true, the framework will look for these two files to be injected and loaded in the runtime bundle of the Next.js App.
+
+```js title=next.config.js
+const { withHeadstartWPConfig } = require('@headstartwp/next/config');
+
+/**
+ * Update whatever you need within the nextConfig object.
+ *
+ * @type {import('next').NextConfig}
+ */
+const nextConfig = {};
+
+module.exports = withHeadstartWPConfig(nextConfig);
+```
+:::caution
+Since `@headstartwp/next@1.2.0` you do not need to import `headstartwp.config.js` in `next.config.js` anymore, the framework will dynamically load the config.
```

Comment:
I think it's fine, `next.config.js` isn't imported anywhere, I'd say engineers should be aware that it is being imported automatically.

Before the recent updates to how the config is injected, `headless.config.js` can be named anything... but now it can only be `headstartwp.config.js` or `headless.config.js` so before this PR someone could think they could name the file anything they want since they are manually importing in `next.config.js` so I think it's less confusing to have it dynamically loaded.

Reply:
Worth noting that if someone is still importing it manually it won't dynamically load.